rmw_fastrtps_cpp/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ find_package(ament_cmake REQUIRED)
 find_package(fastrtps_cmake_module REQUIRED)
 find_package(fastcdr REQUIRED CONFIG)
 find_package(fastrtps REQUIRED CONFIG)
+find_package(FastRTPS REQUIRED MODULE)
```

Comment:
`find_package(fastcdr REQUIRED CONFIG)` and `find_package(fastrtps REQUIRED CONFIG)` use the configurations from `fastcdr` (https://github.com/eProsima/Fast-CDR/blob/master/cmake/packaging/Config.cmake.in) and `fastrtps` (https://github.com/eProsima/Fast-RTPS/blob/master/cmake/packaging/Config.cmake.in) respectively, whereas the added line exercises the module in https://github.com/esteve/rmw_fastrtps/tree/export-libraries/fastrtps_cmake_module/cmake/Modules

The way I see it, either both three lines stay or instead add an `include_directories()` with `${FastRTPS_INCLUDE_DIR}` and add a new `${FastCDR_INCLUDE_DIR}` variable (and add it to this `CMakeLists.txt`) to `FindFastRTPS.cmake`, which would replace the `CONFIG` lines.

What would you suggest? Improve `FindFastRTPS.cmake` and get rid of the `CONFIG` lines or just keep this as is?